### PR TITLE
updated fetch

### PR DESCRIPTION
**Pull request by thisispatrick33**

### CODE REVIEW

**Review by joujo1212 — APPROVED** [review]

File: nuxt.config.ts [hunk 1]
```diff
@@ -13,6 +13,12 @@ export default defineNuxtConfig({
   },
   ssr: false,
   target: 'static',
-  test: true,
+  vite: {
+    server: {
+      watch: {
+        usePolling: true,
```
Comment (peter9ke) [inline nuxt.config.ts:19]: was is das ?

File: machines/fetchTimeSlotsMachine.ts [hunk 2]
```diff
@@ -1,9 +1,17 @@
 import { createMachine, sendParent } from 'xstate'
 import axios from 'axios'
+import { resourcesIDs } from '~/data/resourceIDs'
+import { responseTimeSlotsValidator } from '~/data/timeslotsResponse'
+import { normalizeTimeslots } from '~/utils/appointments'
 
+const PAGE_ID = '6fcb6b4a-d110-46dd-a512-b945b6637b91'
```
Comment (peter9ke) [inline machines/fetchTimeSlotsMachine.ts:7]: toto by sme mali premiestnit niekde inde, lebo to PAGE_ID, budeme potrebovat aj pri inych api calls

File: machines/fetchTimeSlotsMachine.ts [hunk 3]
```diff
@@ -1,9 +1,17 @@
...
 const fetchTimeSlotsMachine = createMachine({
   id: 'fetchTimeSlotsMachine',
   initial: 'idle',
+  context: {
+    startDate: '2022-08-01',
+    endDate: '2022-08-06',
```
Comment (peter9ke) [inline machines/fetchTimeSlotsMachine.ts:13]: toto uz by malo byt dynamicke a nie hardcoded

File: machines/fetchTimeSlotsMachine.ts [hunk 4]
```diff
@@ -13,29 +21,32 @@ const fetchTimeSlotsMachine = createMachine({
...
-              'https://api.booker25.com/api/v2/pages/6fcb6b4a-d110-46dd-a512-b945b6637b91/timeslots/',
+              `https://api.booker25.com/api/v2/pages/${PAGE_ID}/timeslots/`,
               {
-                start: `2022-08-01 00:00`,
-                end: `2022-08-03 23:59`,
+                start: `${context.startDate} 00:00`,
+                end: `${context.endDate} 23:59`,
                 timezone: 'Europe/Berlin',
                 group_size: 1,
                 reservation_type: 'a507S000000PAuKQAW',
```
Comment (peter9ke) [inline machines/fetchTimeSlotsMachine.ts:33]: reservetion_type by malo byt tiez v premennej ulozene

Comment (peter9ke) [reply machines/fetchTimeSlotsMachine.ts:33]:
``` 
{
  name: 'Gratis Beratungstermin',
  reservation_type: 'a507S000000PAuKQAW'
}

File: machines/fetchTimeSlotsMachine.ts (hunk 2 above)
Comment (thisispatrick33) [reply machines/fetchTimeSlotsMachine.ts:7]: Change to PAGE_HAIRSKIN_ID

File: machines/fetchTimeSlotsMachine.ts (hunk 3 above)
Comment (thisispatrick33) [reply machines/fetchTimeSlotsMachine.ts:13]: startDate: current.date,
endDate: CONST = 30 days,

File: machines/fetchTimeSlotsMachine.ts (hunk 4 above)
Comment (thisispatrick33) [reply machines/fetchTimeSlotsMachine.ts:33]: check if it is needed in another files, if not create Constant only to this file

File: machines/fetchTimeSlotsMachine.ts [hunk 5]
```diff
@@ -1,9 +1,22 @@
 import { createMachine, sendParent } from 'xstate'
 import axios from 'axios'
+import { resourcesIDs } from '~/data/resourceIDs'
+import { responseTimeSlotsValidator } from '~/data/timeslotsResponse'
+import { normalizeTimeslots } from '~/utils/appointments'
+import { PAGE_ID_HAIRSKIN, GRATIS_BERATUNGSTERMIN } from '~/utils/variables'
+import { format, addDays } from 'date-fns'
+
+const todayDate = new Date()
+const dateAfterAdding30Days = addDays(todayDate, 30)
```
Comment (peter9ke) [inline machines/fetchTimeSlotsMachine.ts:10]: 30 should be constant

Comment (peter9ke) [reply machines/fetchTimeSlotsMachine.ts:10]: and the name dateAfterAdding30Days should not contain a number

**Pull request merged**